lib/ansible/modules/cloud/google/gcp_compute_vpn_tunnel.py

```diff
@@ -236,6 +236,12 @@
             - Labels to apply to this VpnTunnel.
         returned: success
         type: dict
+    label_fingerprint:
```

Comment:
You might find this upsetting, but unfortunately networking resources _only_ accept labels via SetLabels calls - they do _not_ work on create.